<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocationSelectionWindow</class>
 <widget class="QMainWindow" name="LocationSelectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>490</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 20px;
    background-color: rgba(135, 206, 250, 0.5);
    padding: 10px 20px;
    color: #FFFFFF;
    font-size: 16px;
    border: 2px solid rgba(70, 130, 180, 0.5);
    transition: background-color 0.3s;
}

QPushButton:hover {
    background-color: rgba(135, 206, 250, 0.7); 
}

QPushButton:pressed {
    background-color: rgba(70, 130, 180, 0.7); 
    border: 2px solid rgba(70, 130, 180, 0.7);
}
</string>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="locationSelectionView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>501</width>
      <height>370</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>370</width>
      <height>370</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>501</width>
      <height>370</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="backgroundBrush">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="foregroundBrush">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="interactive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>490</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 20px;
    background-color: rgba(135, 206, 250, 0.5);
    padding: 10px 20px;
    color: #FFFFFF;
    font-size: 16px;
    border: 2px solid rgba(70, 130, 180, 0.5);
    transition: background-color 0.3s;
}

QPushButton:hover {
    background-color: rgba(135, 206, 250, 0.7); 
}

QPushButton:pressed {
    background-color: rgba(70, 130, 180, 0.7); 
    border: 2px solid rgba(70, 130, 180, 0.7);
}
</string>
    </property>
    <property name="text">
     <string>Rotate boats</string>
    </property>
   </widget>
   <widget class="QLabel" name="InformationLabel">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>221</width>
      <height>371</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>221</width>
      <height>371</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Segoe Script</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(135, 206, 250, 0.5);
color: #FFFFFF;</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Place your ships on a 10x10 grid. You have a certain number of ships of different sizes: one four-deck, two three-deck, three two-deck, and four single-deck ships. Arrange them in a way that they do not intersect or overlap. You cannot proceed until all ships are placed.</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
